pkg/storage/mysql/mysql_test.go

```diff
@@ -155,3 +156,27 @@ func TestReadAuthorizationModelUnmarshallError(t *testing.T) {
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "cannot parse invalid wire-format data")
 }
+
+func TestReadUserTupleNullCondition(t *testing.T) {
+	ctx := context.Background()
+	testDatastore := storagefixtures.RunDatastoreTestContainer(t, "mysql")
+
+	uri := testDatastore.GetConnectionURI(true)
+	ds, err := New(uri, sqlcommon.NewConfig())
+	require.NoError(t, err)
+	defer ds.Close()
+
+	stmt := `
+		INSERT INTO tuple (
+			store, object_type, object_id, relation, _user, user_type, ulid,
+			condition_name, condition_context, inserted_at
+		) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, NOW());
+	`
+	ulid := ulid.Make().String()
+	_, err = ds.db.ExecContext(ctx, stmt, "store", "folder", "2021-budget", "owner", "user:anne", "user", ulid, nil, nil)
+	require.NoError(t, err)
+
+	tk := tuple.NewTupleKey("folder:2021-budget", "owner", "user:anne")
+	_, err = ds.ReadUserTuple(ctx, "store", tk)
+	require.NoError(t, err)
```

Comment:
Can we also add an assertion just to make sure the tuples condition matches the normalized spec we expect?